src/project/FileTreeView.js

```diff
@@ -518,6 +518,15 @@ define(function (require, exports, module) {
                 case "gif":
                     fileType = "image";
                     break;
+                case "eot":
+                case "woff":
+                case "otf":
+                case "ttf":
+                    fileType = "font";
+                    break;
+                case "mp3":
```

Comment:
What about `.wav` or `.wave`?  What about `.oga`?  I'm still concerned about how this doesn't align with what's in `content.js`.